styles/style.scss

```diff
@@ -86,6 +86,12 @@ $font-path: '../../fonts';
     .mzp-t-picto-center {
         grid-column: span 3;
 
+        .mzp-c-picto {
+            display: grid;
+            grid-template-rows: 1fr 1fr;
+            height: 100%;
+        }
+
```

Comment:
@reemhamz since the icons aren't all the same size, I made this quick fix to align the pictos, but I don't think it looks that great. Any ideas on how to make this look a little better?

Reply:
For this section, I think best course of action would be to:
1. Increase icon sizes (i.e. `width: 90px`)
2. Align the icons to be in the center rather than the top of their divs, since there is too much spacing between the icons and content
3. Reduce bottom margins

Here is the code I wrote out to get a nice layout:

```suggestion
        .mzp-c-picto {
            display: grid;
            grid-template-rows: 1fr 1fr;
            height: 100%;
            
            .mzp-c-picto-image {
                margin-bottom: 0;
                align-self: center;
            }

            img {
                width: 90px;
                display: block;
            }
        }

```

With the above code, I produced this layout:
<img width="1576" alt="Screen Shot 2022-12-19 at 7 50 19 PM" src="https://user-images.githubusercontent.com/42309026/208555858-369bac81-fe20-4f96-a799-b2a0b8cfd480.png">

Same layout but with borders for clarity on positioning:
<img width="1519" alt="Screen Shot 2022-12-19 at 7 50 57 PM" src="https://user-images.githubusercontent.com/42309026/208555919-6c45783c-9e65-471c-aaea-adf9a494e03a.png">